modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/order/engine/CommerceOrderEngineImpl.java

```diff
@@ -462,7 +474,7 @@ private boolean _isGuestCheckoutEnabled(long groupId) throws Exception {
 			_configurationProvider.getConfiguration(
 				CommerceOrderCheckoutConfiguration.class,
 				new GroupServiceSettingsLocator(
-					groupId, CommerceConstants.SERVICE_NAME_ORDER));
+					groupId, CommerceConstants.ORDER_SERVICE_NAME));
```

Comment:
Hey @alecsloan we should use `SERVICE_NAME_ORDER`. Its just a rename of  `SERVICE_NAME_ORDER`. `SERVICE_NAME_ORDER` will be deprecated.
```suggestion
					groupId, CommerceConstants.SERVICE_NAME_ORDER));
```